### ISSUE TITLE
Please give exportSVG an option to help make unique IDs

### ISSUE THREAD

**Issue by JohnThomson** [paperjs/paper.js#1709]
# Description

When I use exportSVG repeatedly and generate multiple SVGs, various elements in the resulting SVGs can have IDs, for example, <clipPath id="clip-2">...>...<g fill="#ffffff" id="svg8 1" clip-path="url(#clip-2)">. IDs are frequently re-used in distinct SVGs.

If I export, for example, two quite different SVGs that both use clipPath id="clip-2" and then embed them in the SAME HTML document, the result is disastrous. The second SVG is drawn using the clipPath from the first one! This appears to be correct HTML behavior (in the face of incorrect HTML...IDs should be unique): the browser is correctly using the FIRST occurrence of each ID, not in the individual SVG, but in the whole document.

It would be very helpful to have an option on exportSVG to specify an idPrefix, which would be prepended to all the IDs that paper.js would otherwise generate in the SVG. The client then only has to come up with a unique name for each export, rather than having to fix all the IDs individually. Some way of requesting a random ID that is highly unlikely to recur could also help.

# Additional information

I'm generating the SVGs from a program using an embedded version of Firefox version 45. Displaying them either there or in FF 69, on Windows 10, node 10.15.3, paper 0.12.3.

**Comment by lehni** [comment]
Yes I agree, a kind of id prefix / namespace would be useful, and quite easy to add.

**Comment by GlifTek** [comment]
is this available yet?
a way to have <svg id='mySVGid' ...> ??
i can make my items have id's with giving them .name .
why not the top svg tag?
you'd think this would be the most important id of them all.

... also.. is there anyway to NOT include the default activeLayer as a group?
JUST the contents? (no "<g>") ??
